doc/release-process.md

```diff
@@ -212,93 +154,84 @@ Codesigner only: Sign the windows binaries:
+```
 
-- Create `SHA256SUMS.asc` for the builds, and GPG-sign it:
+Here's an equivalent, more readable command if you're confident that you won't
+mess up whitespaces when copy-pasting:
 
 ```bash
-sha256sum * > SHA256SUMS
-```
+cat << EOF > SHA256SUMS.asc
+-----BEGIN PGP SIGNED MESSAGE-----
+Hash: SHA256
 
-The list of files should be:
-```
-bitcoin-${VERSION}-aarch64-linux-gnu.tar.gz
-bitcoin-${VERSION}-arm-linux-gnueabihf.tar.gz
-bitcoin-${VERSION}-riscv64-linux-gnu.tar.gz
-bitcoin-${VERSION}-x86_64-linux-gnu.tar.gz
-bitcoin-${VERSION}-osx64.tar.gz
-bitcoin-${VERSION}-osx.dmg
-bitcoin-${VERSION}.tar.gz
-bitcoin-${VERSION}-win64-setup.exe
-bitcoin-${VERSION}-win64.zip
+$(cat all.SHA256SUMS)
+$(cat all.SHA256SUMS.asc)
+EOF
 ```
-The `*-debug*` files generated by the gitian build contain debug symbols
-for troubleshooting by developers. It is assumed that anyone that is interested
-in debugging can run gitian to generate the files for themselves. To avoid
-end-user confusion about which file to pick, as well as save storage
-space *do not upload these to the bitcoincore.org server, nor put them in the torrent*.
 
-- GPG-sign it, delete the unsigned file:
-```
-gpg --digest-algo sha256 --clearsign SHA256SUMS # outputs SHA256SUMS.asc
-rm SHA256SUMS
-```
-(the digest algorithm is forced to sha256 to avoid confusion of the `Hash:` header that GPG adds with the SHA256 used for the files)
-Note: check that SHA256SUMS itself doesn't end up in SHA256SUMS, which is a spurious/nonsensical entry.
+- Upload to the bitcoincore.org server (`/var/www/bin/bitcoin-core-${VERSION}`):
+    1. The contents of `./bitcoin/guix-build-${VERSION}/output`, except for
+       `*-debug*` files.
 
-- Upload zips and installers, as well as `SHA256SUMS.asc` from last step, to the bitcoincore.org server
-  into `/var/www/bin/bitcoin-core-${VERSION}`
+       The `*-debug*` files generated by the guix build contain debug symbols
+       for troubleshooting by developers. It is assumed that anyone that is
+       interested in debugging can run guix to generate the files for
+       themselves. To avoid end-user confusion about which file to pick, as well
+       as save storage space *do not upload these to the bitcoincore.org server,
+       nor put them in the torrent*.
 
-- A `.torrent` will appear in the directory after a few minutes. Optionally help seed this torrent. To get the `magnet:` URI use:
-```bash
-transmission-show -m <torrent file>
-```
-Insert the magnet URI into the announcement sent to mailing lists. This permits
-people without access to `bitcoincore.org` to download the binary distribution.
-Also put it into the `optional_magnetlink:` slot in the YAML file for
-bitcoincore.org.
+    2. The combined clear-signed message you just created `SHA256SUMS.asc`
+
+- A `.torrent` will appear in the directory after a few minutes. Optionally help
```

Comment:
`bitcoincore.org` does not have functionality to automatically generate torrents. This has been done manually (I use `transmission-gtk`) for a long time.

Reply:
FWIW, I use the following trackers:
```
udp://tracker.openbittorrent.com:80
udp://tracker.opentrackr.org:1337/announce
udp://tracker.coppersurfer.tk:6969/announce

udp://tracker.leechers-paradise.org:6969/announce
udp://explodie.org:6969/announce
udp://tracker.torrent.eu.org:451/announce

udp://tracker.bitcoin.sprovoost.nl:6969
```
After adding @Sjors's tracker, you need to ping him with the magnet hash so he  can whitelist it.
(I don't think these details make sense to document in the release process, it seems to be kind of transient information suspect to change, but just FYI)

Reply:
Good to have these written down even if it's in a review comment :-)
Do you think there's any torrent-related instruction that does belong in release-process.md?

Reply:
Maybe that we create the torrent from the `bitcoin-core-x.x.x` directory (so the directory itself is in the torrent, with in it the archives for every platform as well as the `SHA256SUMS.asc`).
When you do this in `transmission-gtk` it makes a torrent file called `bitcoin-core-x.y.z.torrent` which I've had to rename manually to `bitcoin-x.y.z.torrent` before uploading, as that has been the naming scheme e.g. https://bitcoincore.org/bin/bitcoin-core-0.21.1/bitcoin-0.21.1.torrent.

Somewhat amusing: A commonly asked question is why the `SHA256SUMS.asc` doesn't contain a hash of the torrent, but this is because that would create a chicken-egg problem :smile:

Reply:
Added some info in 181ba751f278f5aac94c5ba1b8beeb6b3ec4629e, let me know if I'm missing anything

Reply:
Looks good to me now!